### DISCUSSION TITLE
Check out MetaDome

### DISCUSSION THREAD

**Discussion by moonso** [Clinical-Genomics/scout#5296] [Ideas]
MetaDome is a tool that scores positions based on how sensitive the position is to changes. Similar to pLi but it uses the information from protein domains combined with variant frequencies. We should investigate if we can score variants  and include in rank model or at least link out to the web based tool.

> MetaDome is a fast and easy-to-use web server that offers an easily accessible and unique perspective that enables researchers and clinicians to interpret variants of unknown significance via 'meta-domains'; a concept that we have previously validated ( L. Wiel et al. Human Mutation, 2017 ) which makes use of protein domain homology within the human genome to transfer genetic variant information across homologous domains. MetaDome can visualize meta-domain information and gene-wide profiles of genetic tolerance.

Browser: https://stuart.radboudumc.nl/metadome/
GitHub: https://github.com/cmbi/metadome

**Comment by dnil** [comment]
Downloadable data is now available! See https://github.com/cmbi/metadome/issues/54#event-6772329823 and https://zenodo.org/record/6625251.

**Reply by dnil** [reply]
So, we actually had some interest in this again - from @fulyataylan. The interface isn't super straightforward to deep link into: js front, calculating on the fly from, I guess the above, a dump of counts of variants in gnomAD, ClinVar and a set of precomputed domains. Sometime cross-site scripting protection is a pain. 😸 Perhaps we revisit the data?

**Reply by dnil** [reply]
So, let's at least aim for a link out first. It is not super easy; will require a two step process where we negotiate with the megadome API on a representative transcript (since we don't carry versioned transcripts in general, and they do). This should be possible though.

And agreed that we should have some more local regional constraint in the scoring model, beyond just the gene. This has support from e.g. ACMG classification guidelines. Metadome could be it, but it hasn't really been updated so as it stands will not carry over to hg38?

**Reply by dnil** [reply]
We can ask this endpoint for a gene:
https://stuart.radboudumc.nl/metadome/api/get_transcripts/POT1
and get a list of supported/potentially supported transcripts and then request a view with the (versioned version) we agree on:
https://stuart.radboudumc.nl/metadome/dashboard/transcript/ENST00000357628.3/

**Reply by fulyataylan** [reply]
I think having a link and the gene name inserted is enough. Then the user can pick the transcript. I think you don't need to spend so much time on the transcript part. I have been using this tool lately and I found it useful to see the constraint parts of the proteins. 

Many thanks @dnil for taking the time to explore this again!

**Reply by dnil** [reply]
And how would you go about doing that?

**Reply by dnil** [reply]
We could help them with a PR for like https://github.com/cmbi/metadome/issues/63 I suppose, but then they have to deploy it etc, and they haven't really updated in a long time...

**Reply by dnil** [reply]
Actually there has been a bit of activity; LvdW is at Stanford now it seems, but has made contributions to https://github.com/laurensvdwiel/MetaDome-vcf-annotation rather recently. Which would fit a pipeline well, and we could pick up the annotations from there.

**Reply by fulyataylan** [reply]
That is great!
